test/e2e/tests/tolerate-failure/onboarding/onboarding-new-wallet.spec.ts

```diff
@@ -0,0 +1,113 @@
+          'Time since the user clicks on "Create password" button until "Recovery Phrase" screen is visible',
+        );
+        const timer4 = Timers.createTimer(
+          'Time since the user clicks on "Skip" button until "Metrics" screen is visible',
+        );
+        const timer5 = Timers.createTimer(
+          'Time since the user clicks on "I agree" button until "Onboarding Success" screen is visible',
+        );
+        const timer6 = Timers.createTimer(
+          'Time since the user clicks on "Done" button until "Home" screen is visible',
+        );
+        const timer7 = Timers.createTimer(
+          'Time since the user clicks on "Done" button until "Home" screen and assets list are visible',
+        );
+
+        await driver.navigate();
+        const startOnboardingPage = new StartOnboardingPage(driver);
+        const metrics = await driver.collectMetrics();
+        console.log('Metrics:', metrics);
+        await startOnboardingPage.checkLoginPageIsLoaded();
+        await startOnboardingPage.createWalletWithSrp(false);
+        timer1.startTimer();
+        await startOnboardingPage.checkSocialSignUpFormIsVisible();
+        timer1.stopTimer();
+
+        await startOnboardingPage.clickCreateWithSrpButton();
+        timer2.startTimer();
+        const onboardingPasswordPage = new OnboardingPasswordPage(driver);
+        await onboardingPasswordPage.checkPageIsLoaded();
+        timer2.stopTimer();
+        await onboardingPasswordPage.createWalletPassword(WALLET_PASSWORD);
+        timer3.startTimer();
+        const secureWalletPage = new SecureWalletPage(driver);
+        await secureWalletPage.checkPageIsLoaded();
+        timer3.stopTimer();
+        await secureWalletPage.skipSRPBackup();
+        timer4.startTimer();
+        const onboardingMetricsPage = new OnboardingMetricsPage(driver);
+        await onboardingMetricsPage.checkPageIsLoaded();
+        timer4.stopTimer();
+        await onboardingMetricsPage.clickOnContinueButton();
+        timer5.startTimer();
+        const onboardingCompletePage = new OnboardingCompletePage(driver);
+        await onboardingCompletePage.checkPageIsLoaded();
+        timer5.stopTimer();
+        await onboardingCompletePage.completeOnboarding();
+        timer6.startTimer();
+        const homePage = new HomePage(driver);
+        await homePage.checkPageIsLoaded();
+        const assetListPage = new AssetListPage(driver);
+        await assetListPage.checkTokenListIsDisplayed();
+        await assetListPage.checkTokenListPricesAreDisplayed();
+        timer6.stopTimer();
+        await homePage.clickBackupRemindMeLaterButton();
+        timer7.startTimer();
+        await homePage.checkPageIsLoaded();
+        await assetListPage.checkTokenListIsDisplayed();
+        await assetListPage.checkTokenListPricesAreDisplayed();
+        await assetListPage.checkTokenExistsInList('Ethereum');
+        await assetListPage.checkTokenExistsInList('Solana');
```

Comment:
same here, this spec fails at this assertion as there's no Solana token/network in the wallet

<img width="1835" height="989" alt="Image" src="https://github.com/user-attachments/assets/431475c5-6390-460d-988d-a97398e642d7" />

Reply:
so this is the same issue I mentioned to Howard in another comment, there is an existing issue (https://github.com/MetaMask/metamask-extension/issues/38104) which is being investigated, that's why the test fails, any of the snap network appear (Solana, BTC and Tron), and when selecting the network, tons of errors show up in the console. 
There are 2 options:
- Comment the Solana check -> Cons: we might forget to enable it back
- Leave it and since it is part of "tolerate-failure" folder, it does not affect CI

For me, the second option is the best one, but open to change it

Reply:
both sound good to me. Thank you for adding the link 🙏